sklearn/linear_model/ridge.py

```diff
@@ -847,8 +847,10 @@ def fit(self, X, y, sample_weight=None):
         -------
         self : Returns self.
         """
+        alphas = np.asarray(self.alphas)
```

Comment:
the tests are passed also without that line.  The problem is that the default argiment was a numpy array. If you want I can delete it and just use self.alphas in all the method.